### PR TITLE
Add 'NTP No Copyright' #947

### PR DESCRIPTION
**Pull request by johnmhoran**
* Added NTP-0.xml and NTP-0.txt.
* Used NTP.xml/txt as templates.

Signed-off-by: John M. Horan <johnmhoran@gmail.com>

### CODE REVIEW

**Comment by johnmhoran** [comment]
Hey @swinslow .  I thought this might go quickly given the relative simplicity of the license text, but I see the test(s) failed.  Will look into that and try to resolve.

**Comment by swinslow** [comment]
Thanks @johnmhoran! If you aren’t sure what the issue is from the test logs, let me know and I’m happy to take a look as well.

**Comment by johnmhoran** [comment]
@swinslow Found it.  I used the existing NTP files as templates and there was a missing "of" in the text (NTP.xml and .txt).  I'd added it to my .xml but forgot to do the same in the .txt test file.  Saving, staging etc. . . .

**Comment by johnmhoran** [comment]
@swinslow   Passed -- excellent.  I look forward to your feedback.  

One possible issue: to address the multi-party references in the sample `e2fsprogs` license, I changed "name" and "make" to "name(s)" and "make(s)".  Is that the proper way to document such alternatives?  

Also, I placed the date placeholder where it is in the sample license, although the NTP license uses a date range at the end of the copyright notice -- do we have a standard way of expressing that?

File: test/simpleTestForGenerator/NTP-0.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+NTP No Copyright (NTP-0)
+
+Copyright (4-digit-year) by (CopyrightHoldersName)
+
+Permission to use, copy, modify, and distribute this software and its documentation for any purpose is hereby granted, provided that the name(s) of  (TrademarkedName) not be used in advertising or publicity pertaining to distribution of the software without specific, written prior permission. (TrademarkedName) make(s) no representations about the suitability of this software for any purpose. It is provided "as is" without express or implied warranty.
```
Comment (waldyrious) [inline test/simpleTestForGenerator/NTP-0.txt]: There are two space characters in `of··(TrademarkedName)`.

Comment (johnmhoran) [reply test/simpleTestForGenerator/NTP-0.txt]: Thanks @waldyrious .  I see that now in my local copy of the .txt file.  I'll go ahead and fix that now.  Out of curiosity, how did you find that, given that the tests passed?

Comment (waldyrious) [reply test/simpleTestForGenerator/NTP-0.txt]: I have the [Refined GitHub](https://github.com/sindresorhus/refined-github) extension installed in my browser, and one of its features is [highlighting whitespace in code and diffs](https://github.com/sindresorhus/refined-github#code), like this:

screenshot

Comment (johnmhoran) [reply test/simpleTestForGenerator/NTP-0.txt]: Very nice -- thank you @waldyrious .

**Pull request merged**